pkg/k8s/watchers/watcher.go

```diff
@@ -450,6 +451,7 @@ var ciliumResourceToGroupMapping = map[string]watcherInfo{
 	synced.CRDResourceName(v2alpha1.CCGName):            {start, k8sAPIGroupCiliumCIDRGroupV2Alpha1},
 	synced.CRDResourceName(v2alpha1.L2AnnouncementName): {skip, ""}, // Handled by L2 announcement directly
 	synced.CRDResourceName(v2alpha1.CPIPName):           {skip, ""}, // Handled by multi-pool IPAM allocator
+	synced.CRDResourceName(v2alpha1.CFLName):            {afterNodeInit, k8sAPIGroupCiliumFlowLog},
```

Comment:
I tried different combinations, but it seems that this patch prevents `cilium-agent` from becoming `Ready`. I can't figure out from the logs alone